backend/app/assets/javascripts/spree/backend/taxonomy.js.coffee

```diff
@@ -116,6 +118,10 @@ root.setup_taxonomy_tree = (taxonomy_id) ->
                 node = m.o
                 new_parent = m.np
 
+                # wait for AJAX requests to finish
+                if $.active > 0
+                  return false
```

Comment:
Weird UX though, if you try and drop a taxon while there's an AJAX request on going it just won't work rather than telling you or showing you why